tests/run_make_tests.pl

```diff
@@ -177,7 +158,7 @@ sub valid_option
 
    if ($option =~ /^-srcdir$/i) {
        $srcdir = shift @argv;
-       if (! -f File::Spec->catfile($srcdir, 'src', 'gnuremake.h')) {
```

Comment:
That's one of the things I didn't quite understand, `catfile` not ncessary anymore

Reply:
I didn't notice this line specificaly, but I _did_ notice in what's up with `$srcdir`. 

What I saw is that if `$srcdir` is set incorrectly, that is, is `src/gnuremake.h` is not around, the `run_make_tests.pl` will refuse proceeed. Given this, I guess Paul Smith though the test was superfluous. 

See also the comment in the code I just made.